### PR TITLE
Rejuvenation

### PR DESCRIPTION
**Pull request by ecoologic**

### CODE REVIEW

File: lib/devise_openid_authenticatable/strategy.rb [hunk 1]
```diff
@@ -100,6 +100,8 @@ def update_resource!(resource)
         resource.openid_fields = fields
       end
 
+      resource.remember_me! if resource.respond_to?(:remember_me!)
+
```
Comment (ecoologic) [inline lib/devise_openid_authenticatable/strategy.rb]: @nbudin This is the solution from @lucascaton (thanks mate) that fixes the most updated build.

File: Gemfile [hunk 2]
```diff
@@ -5,10 +5,11 @@ ruby '2.3.1'
...
-gem 'devise', '~> 3.0.0'
-
 group :test do
-  gem "rots", github: "roman/rots"
-  gem "test-unit" # Needed for github rspec/rspec-rails/issues/1273
+  gem 'rails', '~> 4.0.13'
+  gem 'devise', '~> 3.3.0'
+  gem 'rots', github: 'roman/rots'
+  gem 'test-unit' # Ruby > 2.2 && Rails > 4 https://github.com/rspec/rspec-rails/issues/1273
+  gem 'pry-meta'
```
Comment (ecoologic) [inline Gemfile]: @nbudin do you prefer to remove this dependency?

**Comment by ecoologic** [comment]
I closed it because I'm not sure it works, seems like it resets the token even when it's not needed.

See https://github.com/ecoologic/devise_openid_authenticatable/pull/3

**Comment by nbudin** [comment]
Hi @ecoologic, sorry for not getting around to reviewing this until now.  I think you're right; the solution in this PR is overzealous since it's treating it as if the user has always checked "remember me".

I'm not sure how this is supposed to work in Devise 3.5, but your explanation makes sense to me.

**Comment by ecoologic** [comment]
@nbudin I found it!

https://github.com/ecoologic/devise_openid_authenticatable/pull/3/files#diff-7a1dfc26f4671847a5f40fd9c23c0228R68

I will rebase, doc my commit better and send a new PR.

Basically, on Devise3.3 they were calling remember_me from authenticate, with 3.4 they stopped and only use `success!`.  So I'm overriding it and adding the call to `remember_me` in the same way it was previously called.

**Comment by ecoologic** [comment]
Good news, rails 5 seems pretty easy to update, that will come too (;

**Comment by ecoologic** [comment]
@nbudin rails 5 is also on the way: https://github.com/ecoologic/devise_openid_authenticatable/pull/4

(will also need rebase)

**Pull request closed without merging**